ui/src/templates/constants/defaultTemplates.ts

```diff
@@ -1,41 +1,40 @@
 import {
-  system,
-  localMetrics,
-  gettingStarted,
+  apache,
   docker,
+  gettingStarted,
+  github,
+  kubernetes,
   nginx,
+  ossMetrics,
   redis,
-  kubernetes,
+  system,
 } from '@influxdata/influxdb-templates'
+import {DashboardTemplate} from 'src/types'
 
-export const localMetricsTemplate = () => {
-  return localMetrics
+export const ossMetricsTemplate = (): DashboardTemplate => {
+  return ossMetrics
 }
 
-export const systemTemplate = () => {
-  return system
-}
-
-export const gettingStartedWithFluxTemplate = () => {
-  return gettingStarted
-}
-
-export const staticTemplates = {
-  'getting-started': gettingStarted,
-  'local-metrics': localMetrics,
-  System: system,
+export const staticTemplates: {[k: string]: DashboardTemplate} = {
+  Apache: apache,
   Docker: docker,
-  Redis: redis,
-  Nginx: nginx,
+  'getting-started': gettingStarted,
+  Github: github,
```

Comment:
```suggestion
  GitHub: github,
```
if this key is user facing

Reply:
it's not, but I checked template names and corrected capitalization there.